geometry_msgs/Pose manipulator_pose
geometry_msgs/Pose[] bag_poses
bool setup_environment
string[] bag_labels
string[] bag_domain_labels
string pickup_object
string grasp_type
string drop_object
string planning_domain
bool safe_config
bool go_to_raised
string to_loc
---
bool is_obj_displaced
string held_obj
geometry_msgs/Point displacement
bool success
